src/App.asm

```diff
@@ -1,6 +1,58 @@
 .data
 
+	      str1: .asciiz "Enter total number of students: "
+        str2: .asciiz "Enter student score: "
+        str3: .asciiz "Highest score is: "
+        
 
 .globl main
+.text        
 
-.text
\ No newline at end of file
+main:
+	
+	li $v0, 4
+	la $a0, str1	
+	syscall
+	
+	li $v0, 5
+	syscall
+	move $t0, $v0                   #t0 is  count
+	
+	li $t1, 0 			                #t1 is  i
+	li $t2, -1                      #t2 is  top_student  
+
+	
+loop_begin:
+	blt $t1, $t0,loop_body 
```

Comment:
variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loop_body
...
```